core/src/storage_stage.rs

```diff
@@ -62,7 +63,7 @@ pub const STORAGE_ROTATE_TEST_COUNT: u64 = 128;
 // TODO: some way to dynamically size NUM_IDENTITIES
 const NUM_IDENTITIES: usize = 1024;
 pub const NUM_STORAGE_SAMPLES: usize = 4;
-pub const ENTRIES_PER_SEGMENT: u64 = 16;
+pub const ENTRIES_PER_SEGMENT: u64 = DEFAULT_TICKS_PER_SLOT;
```

Comment:
Not really what this means.

Reply:
These tests are failing even without this change. I changed this hoping it'll fix the test. Let me revert this and push a patch.

Reply:
Thanks @sakridge , looks like this was causing the error.